go.mod

```diff
@@ -29,9 +29,9 @@ require (
 	github.com/gobuffalo/packd v1.0.2
 	github.com/gobuffalo/plush/v4 v4.1.16
 	github.com/goccy/go-yaml v1.9.7
-	github.com/golangci/golangci-lint v1.50.1
+	github.com/golangci/golangci-lint v1.54.2
```

Comment:
Is there any reason to update the `go.mod` in this PR?

Reply:
But let me check again just to make sure i think this might've been from my side!

Reply:
OK it seems that it's working fine now so i reverted my commit.